packages/client/components/TeamPrompt/TeamPromptTopBar.tsx

```diff
@@ -0,0 +1,60 @@
+import graphql from 'babel-plugin-relay/macro'
+import styled from '@emotion/styled'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import useRouter from '~/hooks/useRouter'
+
+import { TeamPromptTopBar_meeting$key } from '~/__generated__/TeamPromptTopBar_meeting.graphql'
+import BackButton from '../BackButton'
+import {HeadingBlock, MeetingTopBarStyles} from '../MeetingTopBar'
+
+const TeamPromptHeaderTitle = styled('h1')({
+  fontSize: 16,
+  lineHeight: '24px',
+  margin: 0,
+  padding: 0
+})
+
+const TeamPromptHeader = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'flex-start'
+})
+
+interface Props {
+  meetingRef: TeamPromptTopBar_meeting$key
+}
+
+const TeamPromptTopBar = (props: Props) => {
+  const {history} = useRouter()
+  const {meetingRef} = props
+
+  const meeting = useFragment(
+    graphql`
+      fragment TeamPromptTopBar_meeting on TeamPromptMeeting {
+        name
+      }
+    `,
+    meetingRef
+  )
+
+  const { name: meetingName } = meeting
+
+  return (
+    <MeetingTopBarStyles>
+      <HeadingBlock>
+        <TeamPromptHeader>
+          <BackButton
+            ariaLabel='Back to Meetings'
+            onClick={() => history.push('/meetings')}
```

Comment:
-1 instead of an `onClick`, can you make `BackButton` extend a `Link` component?

e.g. 
```ts
import {Link} from 'react-router-dom'
const BackButton = styled(Link)({})
```

Reason:
- the `Link` Component uses an anchor tag
- If you right click an anchor tag you'll see "open link in new tab" et al
- that's really nice for a11y, but also in case i want to check out my team dash without leaving the meeting

i just learned this little trick like 2 months ago when some a11y folks told me i need to do better, so the codebase is probably filled with cases where it's not done correctly. if you find other places, please don't hesitate to fix em!

Reply:
Nice tip!  The `BackButton` already internally uses a different component, so I wrapped the existing implementation of `BackButton` with a `Link` and changed it to take a `to` prop passed to `Link` instead of `onChange` passed to `IconButton`